instrumentation/jdbc/src/main/java/io/opentelemetry/instrumentation/auto/jdbc/JdbcTracer.java

```diff
@@ -84,7 +84,6 @@ public Span startSpan(Statement statement, String query) {
     }
 
     DBInfo dbInfo = extractDbInfo(connection);
-
```

Comment:
can u undo this formatting change? only because we are merging into here from otel, so will help avoid merge conflicts